src/App.tsx

```diff
@@ -1,39 +1,89 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import Carousel from './components/Carousel';
+import { images } from './images';
 
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
-  };
-
-  render() {
-    const { images } = this.state;
-
-    return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
+export const App = () => {
+  const [step, setStep] = useState(3);
+  const [frameSize, setFrameSize] = useState(3);
+  const [itemWidth, setItemWidth] = useState(130);
+  const [animationDuration, setAnimationDuration] = useState(1000);
+  const [infinite, setInfinite] = useState(false);
```

Comment:
it's much better to make it as one instance similar you did with movie add form

1. Create a variable outside the Component
```
const initialOptionsState = {
  step: 3,
  frameSize: 3,
  itemWidth: 130,
  animationDuration: 1000,
  infinite: false,
}
```

2. Use one state for options
3. Whenever you need to change value of any field you should pass two arguments `(key, value)` and then just change a value of a movie object by key. In this case you will reuse one handler instead of handlers for each input
Example: 

```
const handleOptionChange = (key, value) => {
  setOptions(prevOptions => ({...prevOptions, [key]: value}))
}
```

```suggestion
  const [options, setOptions] = useState(initialOptionsState);
```